pages/index.vue

```diff
@@ -13,7 +13,7 @@
               <div class="absolute left-0 top-0 flex h-10 w-10 items-center justify-center rounded-lg bg-indigo-600">
                 <component :is="feature.icon" class="h-6 w-6 text-white" aria-hidden="true" />
               </div>
-              {{ feature.name }}
+              <a :href="feature.url">{{ feature.name }}</a>
```

Comment:
curiosity: what is the `:href` doing?

Reply:
the `:` is Vue's way of saying "bind this parameter to the following value" 

so this binds the href parameter to the variable in `feature.url`

Reply:
> the `:` is Vue's way of saying "bind this parameter to the following value"

TIL... - so this is equivalent of

```
<a href="{{ feature.url }}">...
```
?

I'm also surprised by the `"` around `feature.url` - but as I said, I don't know vue.js...